### PR TITLE
PoC GöeIP-001 changes

### PR DESCRIPTION
**Pull request by ayanamidev**
Inflate Testnet ETH to Ninety-two Quintillion Ether according GöeIP-001

https://github.com/goerli/testnet/issues/97

cc @q9f @holiman @karalabe

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -258,16 +259,16 @@ var (
 	//
 	// This configuration is intentionally not using keyed fields to force anyone
 	// adding flags to the config to also have to set these fields.
-	AllEthashProtocolChanges = &ChainConfig{big.NewInt(1337), big.NewInt(0), nil, false, big.NewInt(0), common.Hash{}, big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), big.NewInt(0), nil, nil, new(EthashConfig), nil}
```
Comment (xaiksan1) [inline params/config.go:261]: review

Comment (xaiksan1) [inline params/config.go:261]: commit

**Comment by karalabe** [comment]
This change is gonna be problematic :) Our policy was to only introduce forks needed by mainnet and backport them into the testnets. We never made a testnet specific port and it's a slippery slope from my perspective.

At best, we'd need to figure out a way to spec and apply network specific forks that don't pollute a global namespace.

**Comment by 0xAyanami** [comment]
@karalabe Something like checking the network if it is goerli, and applying those state changes without polluting a global namespace would be still possible however I think it would create some confusion for node operators since the client will show an error as state mismatch, not an hardfork. ( But regarding that this change is a hardfork for goerli )

**Comment by karalabe** [comment]
Going to close this until there's some agreed upon solution. I'm curious if there will be something feasible or we'll only end up launching a replacement testnet. IMHO there's a time when you need to start emptying out tech debs instead of piling it further.

**Pull request closed without merging**